src/Log/LogService.php

```diff
@@ -268,8 +270,14 @@ public function getLogConfig($name)
     {
         $config = $this->getConfig();
 
+        if (! isset($config->logging->log->$name) || ! isset($config->logging->log->$name->format)) {
```

Comment:
Why would we want to do this here? The function was pretty clean. Get default config, then if present get custom config, merge and return. No special treatment of individual properties. I suggest simple deleting the new code here.